malariagen_data/anopheles.py

```diff
@@ -6106,6 +6106,31 @@ def plot_haplotype_clustering(
         df_samples = self.sample_metadata(
             sample_sets=sample_sets, sample_query=sample_query
         )
+
+        debug("Dealing with color")
+        if color and isinstance(color, str):
+            if color not in df_samples.columns and not color.startswith("cohort_"):
+                color = "cohort_" + color
```

Comment:
```suggestion
            if "cohort_" + color in df_samples.columns:
                # Convenience to allow things like "admin1_year" instead of "cohort_admin1_year".
                color = "cohort_" + color
```

Make this a bit more specific.